src/components/Dashboard/AddressListSm.vue

```diff
@@ -290,7 +290,7 @@ $icon-width: 2.75rem;
 }
 
 .v-list-item__subtitle {
-  line-height: 1.25rem;
+  line-height: 1.25rem !important;
```

Comment:
Not sure why this changed but it was OK in Dev and not locally (and I didn't change anything related to this). Anyway, this fixes it.